src/lib/index.jsx

```diff
@@ -68,6 +69,15 @@ class SingleOtpInput extends PureComponent<*> {
 
   getClasses = (...classes) =>
     classes.filter(c => !isStyleObject(c) && c !== false).join(' ');
+  
+  setType = (isSecure, isInputNum) => {
+    if (isSecure) {
+      return 'password'
+    } else 
+    if (isInputNum) {
+      return 'tel'
+    } else return 'text'
+    }
```

Comment:
Nice!

Noticed a few things:
1. Since we're returning from the if blocks, the subsequent "else if" blocks can be replaced with an "if".
2. `isSecure` and `isInputNum` can be read from props here in the function body so passing them as parameters may be omitted.
3. The function can be renamed to getType since it does not actually modify/set the type.

Reply:
Done!